src/app/resperson/index.njk

```diff
@@ -0,0 +1,88 @@
+
+{% block pageTitle %}
+	Manage responsible person | Submit a General Aviation Report (GAR)
+{% endblock %}
+
+{% set active = '/resperson' %}
+
+{% block header %}
+  {% include "header-user.njk" %}
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "phase.njk" %}
+{% endblock %}
+
+{% block content %}
+<script type="text/javascript" src="/javascripts/resperson_utils.js"></script>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% if errors %}
+        {% include "common/templates/includes/errors.njk" %}
+      {% endif %}
+
+      {% if successMsg %}
+        {% include "common/templates/includes/success-action.njk" %}
+      {% endif %}
+
+      <h1 class="govuk-heading-xl">{{ __('heading_responsible_person') }}</h1>
+      <p class="govuk-body-l">{{__('res_person_caption')}}</p>
+      <p class="govuk-body-m">Search for responsible person </p>
+      <input class="govuk-input govuk-input--width-20" id="resperson-search" name="resperson-search" type="text" placeholder="Search" style="margin-bottom: 10px">
+    </div>
+    
+    <div class="govuk-grid-column-full">
+      <p class="govuk-body-s">{{__('person_sort')}}</p>
+      <form id="responsiblePersonId" action="/resperson" method="post">
+        <table class="govuk-table" id="res_person_table">
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <th class="govuk-table__header" id="surname" scope="col">{{__('field_surname')}}</th>
+              <th class="govuk-table__header" id="given_name" scope="col">{{__('field_given_name')}}</th>
+              <th class="govuk-table__header" id="email" scope="col">{{__('field_email')}}</th>
+              <th class="govuk-table__header" scope="col">{{__('table_action')}}</th>
+            </tr>
+          </thead>
+          <tbody class="govuk-table__body" id="resPerson_table_row">
+            {% if resPersons | length %}
+              {% for resPerson in resPersons %}
+                  <tr class="govuk-table__row">
+                    <td class="govuk-table__cell">
+                      <button id="edit-edit_reponsiblePerson-{{ loop.index }}" class="govuk-link btn-link" 
+                      title="Edit details for {{ resPerson.responsibleGivenName }} {{ resPerson.responsibleSurname }}" 
+                      name="editResponsiblePerson" value="{{resPerson.responsiblePersonId}}">{{resPerson.responsibleGivenName}}
+                      </button>
+                    </td>
+                    <td class="govuk-table__cell" scope="row">{{resPerson.responsibleSurname}}</td>
+                    <td class="govuk-table__cell">{{resPerson.responsibleEmail}}</td>
+                    <td class="govuk-table__cell ">
+                      <button id="delete_responsible_person_{{ loop.index }}" class="govuk-link btn-link" name="deleteResponsiblePerson" value="{{resPerson.responsiblePersonId}}">{{__('table_delete')}}</button>
```

Comment:
`Question`: there no confirmation before delete, is this ok for now?
* I am happy with no confirmination, since any more changes seems quite a lot.
* However, it should be mentioned.

Reply:
There are no confirmation before delete as of now, this follows the way as it is done for crafts, people etc. Not sure what is the best approach to show a confirmation message, browsers like Brave can disable the popup confirmation message and can create problem.

Reply:
This is ok.